drivers/volume/volume.go

```diff
@@ -147,6 +147,12 @@ type Driver interface {
 	// GetCSIPodPrefix returns prefix for the csi pod names in the deployment
 	GetCSIPodPrefix() (string, error)
 
+	// PodPrefersBindMount returns true if the Pod prefers local volume attachment
+	PodPrefersBindMount(pod *v1.Pod) bool
```

Comment:
Stork gives equal scores to all nodes with volume replicas irrespective of volume attachment. Even for `ReadWriteOnce` volumes. Maybe this can be done as an improvement to score the node with local volume a higher score. 

PodPrefersBindMount is Portworx's implementation of which Pod prefersBindMount.
I could have just named it IsKubeVirtPod() but we might have cases later where we might want to have this same behavior on other Pods and we will only need to update our implementation of prefersBindMount. prefersBindMount isn't the best name I guess. But wasn't able to come up with a better one. 

PodPrefersBindMount is currently using Pod labels to identify the behavior. for Kubevirt it will be always `ReadWriteMany` volumes. So yes currently this behavior will only get enforced for `ReadWriteMany` volumes.